Fixes rendering issues with django 2.2

karangupta31 opened the pull request:
The issues being faced with the bootstrap not rendering properly seem to be because of the django "aligned" class on fieldsets.
I have added 2 options to the plugin to remove the aligned class from the plugin.

This is a workaround till we don't have a proper solution.

karangupta31 committed:
Fix rendering issue on django 2.2
- Update bootstrap to v 4.5.x
- Add option to mark json editor fieldset independent or shared
- Add css to handle li marker
- Add spectre theme and set as default
- Updated Readme

xrmx commented:
This is more adding a new theme than fixing some rendering issues. Then you are removing django admin classes from javascript which does not seem the proper way to handle that, have you tried overriding stuff in css instead?
BTW have you tried using the `html` theme?

karangupta31 commented:
> This is more adding a new theme than fixing some rendering issues. Then you are removing django admin classes from javascript which does not seem the proper way to handle that, have you tried overriding stuff in css instead?
> BTW have you tried using the `html` theme?

@xrmx 
I added the new theme, because it looked better overall than the bootstrap theme. The html theme looked really bad.

Overriding stuff in css meant, overriding everything django class had under it. I could have changed a few things and made it work for the test scenarios, but i couldn't assure that those 2/3 or 5 overridden styling would help for any new type of property.
Django classes really mess up almost every element from the plugin. 
I tried to be as generic as possible with the solution i implemented. I agree that this is a hack, but its definitely better than the broken state of the package. I'm open to suggestions.

I read about how django form media works and the different properties, but couldn't find anything that might remove the affect of a parent css from the child. The only 2 options i could make out was what i implemented.

karangupta31 commented:
@abogushov Can you please take a look?
